app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,11 @@
+child :content_facet => :content_attributes do
```

Comment:
General comment, do we have a convention that requires the inclusion of '_attributes'?  It seems like it could probably be omitted.  This applies to both subscription_attributes and content_attributes.